Project.toml

```diff
@@ -12,11 +12,13 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Logging = "56ddb016-857b-54e1-b83d-db4d58db5568"
 MacroTools = "1914dd2f-81c6-5fcd-8719-6d5c9610ff09"
 Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
+Memento = "f28f55f0-a522-5efc-85c2-fe41dfb9b2d9"
 NLsolve = "2774e3e8-f4cf-5e23-947b-6d7e65073b56"
 NetworkDynamics = "22e9dc34-2a0d-11e9-0de0-8588d035468b"
 OrderedCollections = "bac558e1-5e72-5ebc-8fee-abe8a469f55d"
 OrdinaryDiffEq = "1dea7af3-3e70-54e6-95c3-0bf5283fa5ed"
 Parameters = "d96e819e-fc66-5662-9728-84c9c7592b0a"
+PowerModelsACDC = "ff45984e-d068-5f4c-9e32-c4133509d236"
```

Comment:
I noticed that you added PowerModelsACDC here but in the script you are using PowerModels.
Which one is correct? 

PS `Ipopt` is missing here, this causes some errors reported in Travis CI